publish-service/src/test/java/com/ericsson/eiffel/remrem/publish/service/MessageServiceRMQImplUnitTest.java

```diff
@@ -86,12 +86,12 @@ public class MessageServiceRMQImplUnitTest {
      * exchange it will throw an Exception.
      */
     @Test
-    public void testCreateExchangeIfNotExistingEnable() throws RemRemPublishException {
+    public void testCreateExchangeIfNotExistingEnable() {
         boolean ExceptionOccured = false;
```

Comment:
ExceptionOccured should be lower-case, e.g. exceptionOccured.

Reply:
changed it accordingly